src/neutral_mixed.cxx

```diff
@@ -31,6 +31,7 @@ NeutralMixed::NeutralMixed(const std::string& name, Options& alloptions, Solver*
   solver->add(Nn, std::string("N") + name);
   solver->add(Pn, std::string("P") + name);
   solver->add(NVn, std::string("NV") + name);
+  /// solver->add(Dnn, std::string("Dnn" + name));
```

Comment:
```suggestion
```

Reply:
Thanks @bendudson Not sure why I left that there, or why Dnnd didn't need to be treated this way. I have a feeling that I tried to add it to match the other variables, but then found that it made Hermes-3 crash. What does `solver->add()` do?

Reply:
Hey @mikekryjak `solver->add()` adds the variable to the time integrator (e.g. CVODE), making it a time-evolving quantity. The solver will set the variable value, and expect `ddt()` of that variable to be set.